src/PHAL_Utilities.hpp

```diff
@@ -94,18 +94,23 @@ class MDFieldIterator {
 template<typename T>
 class MDFieldVectorRight {
 public:
+  using array_type = typename PHX::MDField<T>::array_type;
```

Comment:
This seems unused.

Reply:
Ah actually, looks like it is used in the return type on the next line.

Reply:
Jeez...time to up my prescription glasses...